holidays/countries/south_korea.py

```diff
@@ -625,6 +625,7 @@ class SouthKoreaStaticHolidays:
         2020: (AUG, 17, temporary_public_holiday),
         # Added to create a 6-day long holiday period.
         2023: (OCT, 2, temporary_public_holiday),
+        2024: (OCT, 1, armed_forces_day),
```

Comment:
LGTM, but maybe adding a small footnote here to conform with other existing South Korean 🇰🇷  entries? These wordings are based on https://namu.wiki/w/임시공휴일

```suggestion
        # 76th Anniversary of the Armed Forces of Korea.
        2024: (OCT, 1, armed_forces_day),
```